src/common/components/header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import {Image, Flex, Button, Heading, HStack} from "@chakra-ui/react";
+
+function Header() {
+    return (
+        <HStack bg="white" h="72px" >
+            <Flex justify="space-between" w="100%" >
+                <HStack>
+                    <Image src="https://github.com/UVic-Environmental-Engineering-Club/usv-ground-station/blob/main/public/logo.png?raw=true" alignSelf="center" m="5px 5px 5px 15px" width="50px" height="50px"/>
+                    <Heading m="10px 15px" fontSize={18} > Uvic Environmental Engineering Club</Heading>
```

Comment:
Instead of using `fontSize`, can you use the prop `size`?
https://chakra-ui.com/docs/typography/heading#props

Reply:
Instead of adding 10px from the top, in the parent HStack you can center items vertically